skins/totemearth.scss

```
@@ -0,0 +1,11 @@
+.thirm {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.totemearth {
```